app/javascript/flavours/glitch/features/compose/components/search.jsx

```diff
@@ -360,7 +360,7 @@ class Search extends PureComponent {
 
               <div className='search__popout__menu'>
                 {recent.size > 0 ? this._getOptions().map(({ label, action, forget }, i) => (
-                  <button key={label} onMouseDown={action} className={classNames('search__popout__menu__item search__popout__menu__item--flex', { selected: selectedOption === i })}>
+                  <button key={i} onMouseDown={action} className={classNames('search__popout__menu__item search__popout__menu__item--flex', { selected: selectedOption === i })}>
```

Comment:
Alternatively label + i could be used, but keys are only handled internally by react and ultimately it doesn't matter as long as it's unique.